src/Defs/Alt.cpp

```diff
@@ -1726,6 +1727,31 @@ namespace lua::Class
 		return 1;
 	}
 
+	int Alt::SetMinimapComponentPosition(lua_State* L)
+	{
+		std::string name;
+		char alignX;
+		char alignY;
+		alt::Vector2f pos;
+		alt::Vector2f size;
+		
+		ArgumentReader argReader(L);
+		argReader.ReadString(name);
+		argReader.ReadNumber(alignX);
+		argReader.ReadNumber(alignY);
+		argReader.ReadVector(pos);
+		argReader.ReadVector(size);
+
+		if (argReader.HasAnyError())
+		{
+			argReader.GetErrorMessages();
+			return 0;
+		}
+
+		Core->SetMinimapComponentPosition(name, alignX, alignY, pos, size)
+
+		return 1;	
```

Comment:
if you don't return any value from the stack just use `return 0`, it should match how many values you want to return from the stack